JourneysSquad/JourneysSquad/View/ReusableStyles/CircleWithPlusButton.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+/// Create a button as a circle with a plus in the center.
+struct CircleWithPlusButton: ButtonStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+        Image(systemName: String.AppIconName.plus)
```

Comment:
nit:

Please use the type inferring:

```suggestion
        Image(systemName: .AppIconName.plus)
```